SCOPE.md

```diff
@@ -0,0 +1,53 @@
+# Scope
+
+The REST API specification is meant to provide the functionality currently offer by Wikibase "Action API".
+
+The scope of the schema is limited to the primary entity types (Items & Properties) and the main [CRU(D)](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) actions for those entities.
+
+## Intentionally not implemented
+
+The following elements are considered non essential for the design decisions and are intentionally excluded from the prototype schema:
+
+### Deletions
+
+As entity specific deletion is not yet covered by any Wikibase API the deletion part of CRUD is not covered in this specification.
+
+### Search
+
+A `wbsearchentities` equivalent has not been included in the specification.
+
+Investigations will be carried out to determine if such functionality may make sense to be included as part of the primary entity retrieval endpoint, or if search should remain a separate topic.
+
+### Badges
```

Comment:
Same question as above. [It is currently in the spec](https://github.com/wmde/wikibase-rest-fantasies/blob/54d59b9/paths/index.json#L2). Do we want to remove it or does this instruct people to ignore it?

Reply:
Removal here: https://github.com/wmde/wikibase-rest-fantasies/pull/40